vue/components/ui/molecules/section-expandable/section-expandable.vue

```diff
@@ -96,6 +117,16 @@ export const SectionExpandable = {
             if (this.animated) base.animated = true;
             if (this.expandedData) base.expanded = true;
             return base;
+        },
+        titleClasses() {
+            const base = {};
+            if (this.size) base[`title-${this.size}`] = true;
+            return base;
+        },
+        style() {

```

Comment:
I would rename `style` -> `titleStyle`

Also, as you are not set style with dynamic variables, you should use classes instead, as in, add a `uppercase` class and then have the changes be done via that class activation (like you do above for the title size)